__test__

```diff
@@ -0,0 +1 @@
+***
```

Comment:
Please remove any temporary files in the PR. Change the test so any such files are removed at the end of the test.